fpm-Dockerfile-block-2

```diff
@@ -32,6 +32,7 @@ RUN set -ex \
 	&& { \
 		echo '[global]'; \
 		echo 'daemonize = no'; \
+		echo 'log_limit = 8192'; \
```

Comment:
I agree with @yosifkit; this definitely should be part of `docker.conf` (instead of `zz-docker.conf`) and ought to include a comment pointing to https://github.com/docker-library/php/pull/725#issuecomment-443540114 directly, ala:

```dockerfile
...
 		echo '[global]'; \
 		echo 'error_log = /proc/self/fd/2'; \
 		echo; \
 		echo '; https://github.com/docker-library/php/pull/725#issuecomment-443540114'; \
 		echo 'log_limit = 8192'; \
 		echo; \
...
```

Reply:
done